components/collector/tests/source_collectors/jenkins/test_pipeline_duration.py

```diff
@@ -0,0 +1,57 @@
+"""Unit tests for the Jenkins pipeline duration collector."""
+
+from .base import JenkinsTestCase
+
+
+class JenkinsPipelineDurationTest(JenkinsTestCase):
+    """Unit tests for the Jenkins pipeline duration collector."""
+
+    METRIC_TYPE = "pipeline_duration"
+
+    def setUp(self):
+        """Extend to set up Jenkins data."""
+        super().setUp()
+        self.set_source_parameter("pipeline", "pipeline")
+        self.jenkins_json = {
+            "jobs": [
+                {"name": "main", "builds": [{"result": "FAILURE", "duration": 600_000, "url": "https://failure"}]},
```

Comment:
Why are there underscores here? I don't see that in the jenkins api returns

Reply:
https://docs.python.org/3/reference/lexical_analysis.html#integer-literals